metaflow/plugins/kfp/tests/__init__.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+from typing import List
+from metaflow import R
+
+
+def _python():
+    if R.use_r():
+        return "python3"
+    else:
+        return "python"
+
+
+def obtain_flow_file_paths(flow_dir_path: str, ignore_flows=None) -> List[str]:
```

Comment:
should this be moved to `conftest.py` instead?

Reply:
The list of test file is actually not uniformly defined across tests. 

Only `kfp_flow.py` is ignored for argo test, whereas the existing tests also disables nested foreach tests and treat s3 sensor flows / failure flows differently.